custom/ilsgateway/resources/v0_1.py

```diff
@@ -0,0 +1,26 @@
+from corehq.apps.api.util import get_object_or_not_exist
+from corehq.apps.locations.models import SQLLocation, Location
+from corehq.apps.locations.resources.v0_1 import LocationResource, _user_locations_ids
+
+
+class ILSLocationResource(LocationResource):
+
+    def obj_get_list(self, bundle, **kwargs):
+        domain = kwargs['domain']
+        project = bundle.request.project
+        parent_id = bundle.request.GET.get('parent_id', None)
+        user = bundle.request.couch_user
+        viewable = _user_locations_ids(user, project, only_editable=False)
+
+        if not parent_id:
+            locs = SQLLocation.root_locations(domain, include_archive_ancestors=False)
+        else:
+            parent = get_object_or_not_exist(Location, parent_id, domain)
+            locs = parent.sql_location.get_children().filter(
```

Comment:
looking at the original `LocationResource` it looks like this is the change? I think it'd be easier if you made a function on `LocationResource` called `_base_queryset` or something and then overrode that so you don't have to copy/paste the entire function.